{"uid":"43ac9cd1b536c2ad","name":"test_can_create_connection[Test for Litentry, url: wss://rpc.litentry-parachain.litentry.io]","fullName":"tests.test_nodes_availability#test_can_create_connection","historyId":"9c560a9ef18ba5b1bbc9b73618259a12","time":{"start":1705003534027,"stop":1705003534027,"duration":0},"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:34 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","statusTrace":"request = <SubRequest 'connection_by_url' for <Function test_can_create_connection[Test for Litentry, url: wss://rpc.litentry-parachain.litentry.io]>>\n\n    @pytest.fixture(scope=\"module\", params=collect_nodes_for_chains(get_substrate_chains()), ids=task_ids)\n    def connection_by_url(request):\n        data = request.param\n>       connection = create_connection_by_url(data[\"url\"])\n\ntests/conftest.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nscripts/utils/substrate_interface.py:30: in create_connection_by_url\n    substrate = SubstrateInterface(\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:132: in __init__\n    self.connect_websocket()\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:177: in connect_websocket\n    self.websocket = create_connection(\n.venv/lib/python3.10/site-packages/websocket/_core.py:610: in create_connection\n    websock.connect(url, **options)\n.venv/lib/python3.10/site-packages/websocket/_core.py:255: in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:57: in handshake\n    status, resp = _get_resp_headers(sock)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nsock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>\nsuccess_statuses = (<HTTPStatus.MOVED_PERMANENTLY: 301>, <HTTPStatus.FOUND: 302>, <HTTPStatus.SEE_OTHER: 303>, <HTTPStatus.TEMPORARY_REDIRECT: 307>, <HTTPStatus.PERMANENT_REDIRECT: 308>, <HTTPStatus.SWITCHING_PROTOCOLS: 101>)\n\n    def _get_resp_headers(sock, success_statuses: tuple = SUCCESS_STATUSES) -> tuple:\n        status, resp_headers, status_message = read_headers(sock)\n        if status not in success_statuses:\n            content_len = resp_headers.get('content-length')\n            if content_len:\n                response_body = sock.recv(int(content_len))  # read the body of the HTTP error message response and include it in the exception\n            else:\n                response_body = None\n>           raise WebSocketBadStatusException(\"Handshake status {status} {message} -+-+- {headers} -+-+- {body}\".format(status=status, message=status_message, headers=resp_headers, body=response_body), status, status_message, resp_headers, response_body)\nE           websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:34 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'\n\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:150: WebSocketBadStatusException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":2,"retriesStatusChange":false,"beforeStages":[{"name":"connection_by_url","time":{"start":1705003502162,"stop":1705003503091,"duration":929},"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:03 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 23, in connection_by_url\n    connection = create_connection_by_url(data[\"url\"])\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/scripts/utils/substrate_interface.py\", line 30, in create_connection_by_url\n    substrate = SubstrateInterface(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 132, in __init__\n    self.connect_websocket()\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 177, in connect_websocket\n    self.websocket = create_connection(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 610, in create_connection\n    websock.connect(url, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 255, in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 57, in handshake\n    status, resp = _get_resp_headers(sock)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 150, in _get_resp_headers\n    raise WebSocketBadStatusException(\"Handshake status {status} {message} -+-+- {headers} -+-+- {body}\".format(status=status, message=status_message, headers=resp_headers, body=response_body), status, status_message, resp_headers, response_body)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"connection_by_url","time":{"start":1705003518166,"stop":1705003518961,"duration":795},"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:18 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 23, in connection_by_url\n    connection = create_connection_by_url(data[\"url\"])\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/scripts/utils/substrate_interface.py\", line 30, in create_connection_by_url\n    substrate = SubstrateInterface(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 132, in __init__\n    self.connect_websocket()\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 177, in connect_websocket\n    self.websocket = create_connection(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 610, in create_connection\n    websock.connect(url, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 255, in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 57, in handshake\n    status, resp = _get_resp_headers(sock)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 150, in _get_resp_headers\n    raise WebSocketBadStatusException(\"Handshake status {status} {message} -+-+- {headers} -+-+- {body}\".format(status=status, message=status_message, headers=resp_headers, body=response_body), status, status_message, resp_headers, response_body)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"connection_by_url","time":{"start":1705003534027,"stop":1705003534798,"duration":771},"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:34 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 23, in connection_by_url\n    connection = create_connection_by_url(data[\"url\"])\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/scripts/utils/substrate_interface.py\", line 30, in create_connection_by_url\n    substrate = SubstrateInterface(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 132, in __init__\n    self.connect_websocket()\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/substrateinterface/base.py\", line 177, in connect_websocket\n    self.websocket = create_connection(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 610, in create_connection\n    websock.connect(url, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_core.py\", line 255, in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 57, in handshake\n    status, resp = _get_resp_headers(sock)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/websocket/_handshake.py\", line 150, in _get_resp_headers\n    raise WebSocketBadStatusException(\"Handshake status {status} {message} -+-+- {headers} -+-+- {body}\".format(status=status, message=status_message, headers=resp_headers, body=response_body), status, status_message, resp_headers, response_body)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:34 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","statusTrace":"request = <SubRequest 'connection_by_url' for <Function test_can_create_connection[Test for Litentry, url: wss://rpc.litentry-parachain.litentry.io]>>\n\n    @pytest.fixture(scope=\"module\", params=collect_nodes_for_chains(get_substrate_chains()), ids=task_ids)\n    def connection_by_url(request):\n        data = request.param\n>       connection = create_connection_by_url(data[\"url\"])\n\ntests/conftest.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nscripts/utils/substrate_interface.py:30: in create_connection_by_url\n    substrate = SubstrateInterface(\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:132: in __init__\n    self.connect_websocket()\n.venv/lib/python3.10/site-packages/substrateinterface/base.py:177: in connect_websocket\n    self.websocket = create_connection(\n.venv/lib/python3.10/site-packages/websocket/_core.py:610: in create_connection\n    websock.connect(url, **options)\n.venv/lib/python3.10/site-packages/websocket/_core.py:255: in connect\n    self.handshake_response = handshake(self.sock, url, *addrs, **options)\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:57: in handshake\n    status, resp = _get_resp_headers(sock)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nsock = <ssl.SSLSocket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>\nsuccess_statuses = (<HTTPStatus.MOVED_PERMANENTLY: 301>, <HTTPStatus.FOUND: 302>, <HTTPStatus.SEE_OTHER: 303>, <HTTPStatus.TEMPORARY_REDIRECT: 307>, <HTTPStatus.PERMANENT_REDIRECT: 308>, <HTTPStatus.SWITCHING_PROTOCOLS: 101>)\n\n    def _get_resp_headers(sock, success_statuses: tuple = SUCCESS_STATUSES) -> tuple:\n        status, resp_headers, status_message = read_headers(sock)\n        if status not in success_statuses:\n            content_len = resp_headers.get('content-length')\n            if content_len:\n                response_body = sock.recv(int(content_len))  # read the body of the HTTP error message response and include it in the exception\n            else:\n                response_body = None\n>           raise WebSocketBadStatusException(\"Handshake status {status} {message} -+-+- {headers} -+-+- {body}\".format(status=status, message=status_message, headers=resp_headers, body=response_body), status, status_message, resp_headers, response_body)\nE           websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:34 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'\n\n.venv/lib/python3.10/site-packages/websocket/_handshake.py:150: WebSocketBadStatusException","steps":[],"attachments":[{"uid":"769ecbb0dfa1f7a5","name":"stdout","source":"769ecbb0dfa1f7a5.txt","type":"text/plain","size":1473}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_nodes_availability"},{"name":"host","value":"fv-az735-978"},{"name":"thread","value":"2075-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_nodes_availability"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"connection_by_url","value":"{'url': 'wss://rpc.litentry-parachain.litentry.io', 'name': 'Litentry'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"2d54fd3476d66afc","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:18 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1705003518166,"stop":1705003518166,"duration":0}},{"uid":"8accd678627d2d6e","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 20:05:03 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1705003502161,"stop":1705003502161,"duration":0}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":9,"skipped":0,"passed":700,"unknown":0,"total":709},"items":[{"uid":"1f59ff857db50197","reportUrl":"https://novasamatech.github.io/test-runner/4494//#testresult/1f59ff857db50197","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 16:05:45 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1704989144971,"stop":1704989144971,"duration":0}},{"uid":"e2dfbfe21f400f43","reportUrl":"https://novasamatech.github.io/test-runner/4493//#testresult/e2dfbfe21f400f43","status":"passed","time":{"start":1704974740497,"stop":1704974740497,"duration":0}},{"uid":"231f20739812125f","reportUrl":"https://novasamatech.github.io/test-runner/4492//#testresult/231f20739812125f","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Thu, 11 Jan 2024 08:05:09 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1704960308264,"stop":1704960308264,"duration":0}},{"uid":"f53954e204416afb","reportUrl":"https://novasamatech.github.io/test-runner/4491//#testresult/f53954e204416afb","status":"passed","time":{"start":1704945898928,"stop":1704945898928,"duration":0}},{"uid":"cadb0e8058bf855c","reportUrl":"https://novasamatech.github.io/test-runner/4490//#testresult/cadb0e8058bf855c","status":"passed","time":{"start":1704931570205,"stop":1704931570205,"duration":0}},{"uid":"b4e09cdc992f73c1","reportUrl":"https://novasamatech.github.io/test-runner/4489//#testresult/b4e09cdc992f73c1","status":"passed","time":{"start":1704917126283,"stop":1704917126283,"duration":0}},{"uid":"f059822cec1cdb16","reportUrl":"https://novasamatech.github.io/test-runner/4488//#testresult/f059822cec1cdb16","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Wed, 10 Jan 2024 16:05:22 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1704902721037,"stop":1704902721037,"duration":0}},{"uid":"f1661162177153e5","reportUrl":"https://novasamatech.github.io/test-runner/4487//#testresult/f1661162177153e5","status":"passed","time":{"start":1704888258216,"stop":1704888258216,"duration":0}},{"uid":"6ff8af6729614a2e","reportUrl":"https://novasamatech.github.io/test-runner/4486//#testresult/6ff8af6729614a2e","status":"passed","time":{"start":1704873877430,"stop":1704873877430,"duration":0}},{"uid":"70aff3f373190a62","reportUrl":"https://novasamatech.github.io/test-runner/4485//#testresult/70aff3f373190a62","status":"passed","time":{"start":1704859459074,"stop":1704859459074,"duration":0}},{"uid":"9e7c87eeb63efbf7","reportUrl":"https://novasamatech.github.io/test-runner/4484//#testresult/9e7c87eeb63efbf7","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Wed, 10 Jan 2024 00:05:42 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1704845141618,"stop":1704845141618,"duration":0}},{"uid":"90f010bf0f2bc008","reportUrl":"https://novasamatech.github.io/test-runner/4483//#testresult/90f010bf0f2bc008","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Tue, 09 Jan 2024 20:05:14 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1704830713881,"stop":1704830713881,"duration":0}},{"uid":"7f9f0b5108977b50","reportUrl":"https://novasamatech.github.io/test-runner/4482//#testresult/7f9f0b5108977b50","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Tue, 09 Jan 2024 16:05:27 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1704816326205,"stop":1704816326205,"duration":0}},{"uid":"b5308604dcaf8fe2","reportUrl":"https://novasamatech.github.io/test-runner/4481//#testresult/b5308604dcaf8fe2","status":"broken","statusDetails":"websocket._exceptions.WebSocketBadStatusException: Handshake status 500 Internal Server Error -+-+- {'date': 'Tue, 09 Jan 2024 12:05:32 GMT', 'content-type': 'text/html', 'content-length': '177', 'connection': 'keep-alive', 'server': 'nginx/1.20.1'} -+-+- b'<html>\\r\\n<head><title>500 Internal Server Error</title></head>\\r\\n<body>\\r\\n<center><h1>500 Internal Server Error</h1></center>\\r\\n<hr><center>nginx/1.20.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'","time":{"start":1704801931568,"stop":1704801931568,"duration":0}},{"uid":"9f95100ea838c2fd","reportUrl":"https://novasamatech.github.io/test-runner/4406//#testresult/9f95100ea838c2fd","status":"passed","time":{"start":1703750655962,"stop":1703750655962,"duration":0}},{"uid":"9c58880dca66864c","reportUrl":"https://novasamatech.github.io/test-runner/4405//#testresult/9c58880dca66864c","status":"passed","time":{"start":1703736254901,"stop":1703736254901,"duration":0}},{"uid":"2909ba4fc59a8301","reportUrl":"https://novasamatech.github.io/test-runner/4404//#testresult/2909ba4fc59a8301","status":"passed","time":{"start":1703721917213,"stop":1703721917213,"duration":0}},{"uid":"6beb5f62b451e0a6","reportUrl":"https://novasamatech.github.io/test-runner/4403//#testresult/6beb5f62b451e0a6","status":"passed","time":{"start":1703707534228,"stop":1703707534229,"duration":1}},{"uid":"2e742a73108078c2","reportUrl":"https://novasamatech.github.io/test-runner/4402//#testresult/2e742a73108078c2","status":"passed","time":{"start":1703693086358,"stop":1703693086358,"duration":0}},{"uid":"1fa48f349ba3814b","reportUrl":"https://novasamatech.github.io/test-runner/4401//#testresult/1fa48f349ba3814b","status":"passed","time":{"start":1703678665672,"stop":1703678665672,"duration":0}}]},"tags":[]},"source":"43ac9cd1b536c2ad.json","parameterValues":["{'url': 'wss://rpc.litentry-parachain.litentry.io', 'name': 'Litentry'}"]}